tests/test_multichannel.py

```diff
@@ -201,7 +203,64 @@ def test_tempo_multi(y_multi):
 
     # Check that they're not both the same
     assert not np.allclose(t0, t1)
-    print(t,t0,t1)
+
+
```

Comment:
PLP Implementation looks good!

Some notes on the test:

1. It looks like `ctx` is always `dnr` (do not raise) here, in which case we should be able to skip it entirely.
2. One hop and window is sufficient here, no use in burning cycles for things we already test elsewhere
3. `use_onset` parameter is not used in the test.  I don't think it's critical that we test multi-channel with and without onset envelope construction since the code path is identical and we test that elsewhere.  May as well remove that parameter.

Reply:
Thanks!
    
    1. I was trying other parameter combinations but I removed some, and now there's only `dnr()`
    2. alright
    3.  same story as 1

Pushing a commit with fixes now.